packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2917,6 +2918,8 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
           fontWeight: _style.fontWeight,
           textDirection: _textDirection,
           textAlign: widget.textAlign,
+          lineHeight: _style.height,
+          letterSpacing: _style.letterSpacing,
```

Comment:
Do these translate directly to their CSS equivalents? In my basic testing, it seems to work out fine - however, our y-translation from the transforms we apply end up being off by a couple of pixels

Reply:
Replied to this in https://github.com/flutter/flutter/pull/135163#issuecomment-1758654663